src/ContainerInstance/test/Invoke-AzContainerInstanceCommand.Recording.json

```diff
@@ -1,38 +0,0 @@
-{
-  "Invoke-AzContainerInstanceCommand+[NoContext]+ExecuteExpanded+$POST+https://management.azure.com/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/test-rgdabsp/providers/Microsoft.ContainerInstance/containerGroups/bez-test-cg/containers/bez-test-ci/exec?api-version=2021-09-01+1": {
-    "Request": {
-      "Method": "POST",
-      "RequestUri": "https://management.azure.com/subscriptions/0b1f6471-1bf0-4dda-aec3-cb9272f09590/resourceGroups/test-rgdabsp/providers/Microsoft.ContainerInstance/containerGroups/bez-test-cg/containers/bez-test-ci/exec?api-version=2021-09-01",
-      "Content": "{\r\n  \"terminalSize\": {\r\n    \"rows\": 12,\r\n    \"cols\": 12\r\n  },\r\n  \"command\": \"echo hello\"\r\n}",
-      "isContentBase64": false,
-      "Headers": {
-      },
-      "ContentHeaders": {
-        "Content-Type": [ "application/json" ],
-        "Content-Length": [ "91" ]
-      }
-    },
```

Comment:
Could you check why this recording file is removed

Reply:
There's no corresponding test for this command. I have not deleted it in my current PR. That is why when I ran test module, it removed this file.